src/dict/dictionary.hh

```diff
@@ -316,7 +316,7 @@ protected:
   bool synonymSearchEnabled;
   string dictionaryName;
   //default to true;
-  bool enable_FTS = true;
+  std::optional< bool > enable_FTS;
```

Comment:
Comment is wrong now. It should be changed to "The enable_fts option in metadata.toml"

I think enable_FTS should be renamed to metadata_enable_FTS.

As of now, the 3 variables 

* can_FTS
* fts.enabled
* enable_FTS

have no obvious difference.

Reply:
To properly initialize `std::optional< bool > metadata_enable_FTS = std::nullopt`